src/schema/nmdc.yaml

```diff
@@ -2873,3 +2874,16 @@ slots:
     contributors:
       - ORCID:0009-0001-1555-1601 #Anastasiya Prymolenna
       - ORCID:0000-0002-8683-0050 #Montana Smith
+
+  bulk_elect_conductivity:
+    comments:
+      - Compatible with EMSL metadata
+    description: 
+      Electrical conductivity is a measure of the bulk soil ability to carry electric 
+      current, which is mostly dictated by the chemistry of and amount of soil water. 
+      Provide the value output of the field instrument.
```

Comment:
well written `description`. are we sure that `bulk_elect_conductivity` only applies to soil samples?

instructions to users shouldn't be included in `description`.  `comments` would probably be better.

Reply:
I am not sure! Let's take out soil and just say measure of the sample's ability

Reply:
Changed language to be general for sample type and moved instruction sentence to comments